syft/execution/role.py

```diff
@@ -171,14 +171,39 @@ def execute(self):
 
         return tuple(p.child for p in output_placeholders)
 
+    def store(self, tensor):
```

Comment:
I'm still not sure about this will be used but would we also need something to update the value of an object?
The "usecase" would be: at the beginning of the protocol, I use `load` and at the end I update the initial value with the one we have computed during the protocol.
Maybe that needs discussion though

Reply:
Imagine `Role` has its own `ObjectStore`, which works like a separate "namespace" that holds inputs, intermediate results, and outputs. Two possibilities for how `load/store` could work:
* `load/store` transfer or copy tensors from a `Worker's` `ObjectStore` to a `Role's` `ObjectStore` for inputs and transfer/copy the outputs back
* `load` "reads" inputs from a `Role's` `ObjectStore` (expecting them to be pre-populated) and "writes" outputs back to the `ObjectStore` (expecting them to be extracted afterward.

Reply:
Oh, that's clearer, thanks!
So in my opinion, we do need an update feature, maybe in another method?

Reply:
`update` would be like an in-place operation where the new value overwrites the old one?

Reply:
add a storage_id kwargs so you can "over-write" and update. let me know how it looks.